login.html

```diff
@@ -0,0 +1,14 @@
+<div id="navbar" class="navbar-collapse collapse">
+  <div class="navbar-right user-info">Welcome, <span class="user-fullname">First Last</span> <a class="btn btn-default btn-sm">Logout</a></div>
+  <form class="navbar-form navbar-right" role="form" id="login-form">
+    <div class="form-group">
+      <input type="text" placeholder="Email" class="form-control">
+    </div>
+    <div class="form-group">
+      <input type="password" placeholder="Password" class="form-control">
+    </div>
+    <a class="btn btn-default btn-sm">Login</a>
+  </form>
+</div>
+</div>
```

Comment:
I couldn't find the matching <div> and <nav> tags for these last two items (line 13 and 14). Those may need to be removed? Other than that this code looks great and I vote we merge.